lib/scout/view/mobile_scout.dart

```diff
@@ -0,0 +1,680 @@
+                                      isUline: false,
+                                    ),
+                                  ),
+                                  // sport filter dropdown
+                                  SizedBox(
+                                    child: PopupMenuButton(
+                                      // dropdown options
+                                      itemBuilder: (context) => [
+                                        PopupMenuItem(
+                                          value: 1,
+                                          child: ListTile(
+                                            title: Row(
+                                              mainAxisAlignment:
+                                                  MainAxisAlignment.center,
+                                              children: [
+                                                Text(
+                                                  'All Sports',
+                                                  style: textSwapState(
+                                                    condition: allSportsTitle ==
+                                                        'All Sports',
+                                                    tabNotSelected: textStyle(
+                                                      Colors.white,
+                                                      sportFilterTxSz,
+                                                      isBold: false,
+                                                      isUline: false,
+                                                    ),
+                                                    tabSelected: textStyle(
+                                                      Colors.amber[400]!,
+                                                      sportFilterTxSz,
+                                                      isBold: false,
+                                                      isUline: true,
+                                                    ),
+                                                  ),
+                                                ),
+                                              ],
+                                            ),
+                                          ),
+                                          onTap: () {
+                                            myController.clear();
+                                            if (allSportsTitle !=
+                                                'All Sports') {
+                                              setState(
+                                                () {
+                                                  allSportsTitle = 'All Sports';
+                                                  _selectedSport =
+                                                      SupportedSport.all;
+                                                },
+                                              );
```

Comment:
after `setState` write this:
`context.read<ScoutPageBloc>().add(
                                    const SelectedSportChanged(
                                      selectedSport: SupportedSport.all,
                                    ),
                                  );`
SInce this will filter the sports by the 'all' category